### ISSUE TITLE
print_stats not working on jupyter notebook 

### ISSUE THREAD

**Issue by elyase** [vpelletier/pprofile#16]
```python
>>> profiler.print_stats()
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-229-57c24eb824a6> in <module>()
----> 1 profiler.print_stats()

/Users/elyase/miniconda3/envs/test/lib/python3.5/site-packages/pprofile.py in print_stats(self)
    518         Returns None.
    519         """
--> 520         self.annotate(_reopen(sys.stdout, errors='replace'))
    521 
    522 class ProfileRunnerBase(object):

/Users/elyase/miniconda3/envs/test/lib/python3.5/site-packages/pprofile.py in _reopen(stream, encoding, errors)
     64         # Also, I do not expect many 3.0 and 3.1 to be still used. Feel free to
     65         # report a bug if it raises.
---> 66         return codecs.getwriter(encoding)(stream.buffer, errors=errors)
     67 
     68 def _getFuncOrFile(func, module, line):

AttributeError: 'OutStream' object has no attribute 'buffer'
```

**Comment by vpelletier** [comment]
Hello.

THis should be improved by 93a80a32e4ba1837b74886d8e1d043338aacd4cc . At the expense of reopening the possibility of later encoding error when/if annotating source files which would contain entities invalid in `sys.stdout` encoding (which is what `_reopen` is about). I see ipykernel.iostream.OutStream has encoding set to UTF-8, so it should hopefully not cause issues there.

Could you try that modification ?

**Comment by vpelletier** [comment]
Tested & released as 1.10.0 .
